extensions/amp-access/0.1/amp-access.js

```diff
@@ -351,21 +273,16 @@ export class AccessService {
 
   /** @private */
   startInternal_() {
-    dev().fine(TAG, 'config:', this.type_, this.loginConfig_,
-        this.adapter_.getConfig());
-
     // TODO(dvoytenko, #3742): This will refer to the ampdoc once AccessService
     // is migrated to ampdoc as well.
     Services.actionServiceForDoc(this.ampdoc).installActionHandler(
         this.accessElement_, this.handleAction_.bind(this));
 
-    // Calculate login URLs right away.
-    this.buildLoginUrls_();
-
-    // Start sign-in.
-    this.signIn_.start();
+    for (let i = 0; i < this.sources_.length; i++) {
+      this.sources_[i].start();
```

Comment:
This is actually a pretty critical piece. The `Source.start()` currently starts authorization w/o blocking on document's firstVisible. We can't do this at this time. Any authorization must be predicated by first visibility.

Reply:
oops! great catch, i think i botched this in a refactor. fixed now.